voidcrew/code/game/objects/structures/tables_racks.dm

```diff
@@ -25,20 +18,21 @@
 	else
 		return "<span class='notice'>The top cover is firmly stuck on, but could be <i>pried</i> off with considerable effort.</span>"
 
-/obj/structure/table/wood/reinforced/attackby(obj/item/W, mob/user, params)
-	if(W.tool_behaviour == TOOL_CROWBAR && user.a_intent != INTENT_HELP)
-		if(!W.tool_start_check(user, amount=0))
-			return
+/obj/structure/table/wood/reinforced/crowbar_act(mob/living/user, obj/item/item)
+	. = ..()
+	if (user.a_intent == INTENT_HELP)
+		return
+	if (!item.tool_start_check(user, amount = 0))
+		return
 
-		if(deconstruction_ready)
-			to_chat(user, "<span class='notice'>You begin levering the top cover back in place...</span>")
-			if (W.use_tool(src, user, 50, volume=50))
-				to_chat(user, "<span class='notice'>You pry the top cover back into place.</span>")
-				deconstruction_ready = 0
-		else
-			to_chat(user, "<span class='notice'>You start prying off the top cover...</span>")
-			if (W.use_tool(src, user, 50, volume=50))
-				to_chat(user, "<span class='notice'>You pry off the top cover.</span>")
-				deconstruction_ready = 1
+	if(deconstruction_ready)
+		to_chat(user, "<span class='notice'>You begin levering the top cover back in place...</span>")
+		if (item.use_tool(src, user, 50, volume = 50))
+			to_chat(user, "<span class='notice'>You pry the top cover back into place.</span>")
+			deconstruction_ready = 0
 	else
-		. = ..()
+		to_chat(user, "<span class='notice'>You start prying off the top cover...</span>")
+		if (item.use_tool(src, user, 50, volume = 50))
+			to_chat(user, "<span class='notice'>You pry off the top cover.</span>")
+			deconstruction_ready = 1
```

Comment:
small nitpick that this is TRUE/FALSE, also this is copy paste code from fucking reinforced tables so I hate this.

Reply:
ok im repathing this in separate PR